content.md

```diff
@@ -296,6 +296,9 @@ Some cards such as action cards are not resolved immediately; in that case, they
 > another card. Even drawing to replace cards, comes after this time
 > period.
 
+Cards effects resolve in full: they are not interruptible nor pre-emptible, and their effects cannot be "split up" in a sequence unless they are in an explicit sequence.
```

Comment:
```suggestion
Any effect can only be used if all of its requirements are met. Likewise, the resolution of a single effect always resolves in full. They are not interruptible nor pre-emptible, and their resolution cannot be split unless they are in an explicit sequence. 
```

I tried to clarify a bit, also that you need to have an equipment / retainer in hand if Disguised Weapon / Pack Alpha is used to bring them into play. With this change you shouldn't be able to DI the equipment, only "Disguised Weapon" as well.

The weapon must be announced and in hand when the card is played, before it is replaced, but need not to be shown: it can be kept apart face down. 

[[LSJ 19991207]](https://groups.google.com/g/rec.games.trading-cards.jyhad/c/aQaOTYwC-fg/m/vdyo09QZoicJ) 
[[ANK 20180925-2]](https://www.vekn.net/forum/rules-questions/77029-order-of-draw-and-replace-for-concealed-weapon-under-infernal-pursuit?start=6#90757) 
[[LSJ 20100130]](https://groups.google.com/g/rec.games.trading-cards.jyhad/c/X8Uu7Sk56P4/m/twUvR4ZnBBkJ)